schemas/cgTranscripts.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var cgTranscriptSchema = new Schema({
+  bitsId: String,
+  date: Date,
```

Comment:
use this in place of date , 
```
applyDate: {
type: Date,
default: new Date()
}
```

Reply:
Or event better, use the timestamps provided by mongoose. Refer https://mongoosejs.com/docs/guide.html#timestamps